house_style_guide/house_style_guide.rst

```diff
@@ -0,0 +1,457 @@
+If expressing a number greater than 999 in numerals, use a comma.
+
+- 5 billion people
+- 1,200 years ago
+
+Millions and Billions
+~~~~~~~~~~~~~~~~~~~~~
+
+When stating million or billion with a numeral, don't hyphenate, even before a noun. 
+But do use a hyphen between the numeral and million or billion if the expression is part of a compound adjective that takes a hyphen elsewhere.
+
+- A $6 million lawsuit
+- The 400-million-served mark
+
+If space is tight (for example, in headlines, tables, diagrams, or text messages), some abbreviations are acceptable.
+
+- mil (million). see "mil."
+- bil (billion). see "bil."
+- k (thousand) - Among other things, K can stand for thousand, kilobytes, kilobits, and kilograms. Use K only if its meaning is clear from the context.
+
+At the Beginning of a Sentence
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Avoid starting a sentence with a numeral. 
+If you can't avoid it, spell out the number.
+
+- 450 gamers participated in last night's chat. (don't)
+- Four hundred and fifty gamers participated in last night's chat.
+- Last night, 450 gamers participated in the chat.
+
+A year may be written in numerals at the beginning of a sentence. 
+It's okay to start a headline with a numeral if space is tight or if the numeral makes the headline more eye-catching or easier to scan or understand.
+
+- 1967 was the Summer of Love in San Francisco.
+- 8 Diet Tips
+- Two 5-Year-Old Boys Found; One Still Missing
+
+Ordinals and Suffixes
+~~~~~~~~~~~~~~~~~~~~~
+
+Avoid expressing ordinals with superscript letters (such as 10th, 11th, and so on). 
+Ordinals with superscript formatted in a word processor may not display correctly in some places, such as email, and the use in writing of these suffixes is becoming increasingly archaic. 
+Do not use "th" endings or their equivalent when stating dates (i.e., do not write Join us on June 10th).
+
+Attribution
+-----------
+
+It's important to properly cite sources in all ownCloud documentation. 
+If you've got a case not covered here, find a way to attribute your source that seems appropriate to the medium. 
+Choosing not to cite your source is an unacceptable alternative.
+
+Link Attribution
+~~~~~~~~~~~~~~~~
+
+When including links in external content such as blog posts and ebooks, use anchor text when possible. 
+Linking to other articles and websites is a best practice.
+
+- ownCloud the open platform for more productivity and security in digital collaboration.
+- Check out owncloud.com for more information. (*When including links in presentations, you can use the domain as anchor text*.)
+
+Image Attribution
+~~~~~~~~~~~~~~~~~
+
+When using photos or images from Flickr or another online source under the creative commons license, they must be attributed. 
+When attributing an image, please use the following format and link back to the original source image using the photographer's name as anchor text.
+
+- Image Credit: linked source
+
+Commonly Troublesome Words
```

Comment:
Would that also be a section for common misunderstandings in ownCloud concepts ?

Nothing concrete falls in mind but for example "when talking about an app always say ownCloud server app" to distinguish it from client/mobile apps. Don't use this as I think it's not really needed.